lib/go-tc/enum.go

```diff
@@ -55,6 +55,9 @@ type DeliveryServiceName string
 // CacheType is the type (or tier) of a CDN cache.
 type CacheType string
 
+// InterfaceName is the name of the interface
```

Comment:
This doesn't help a user who doesn't already know what interfaces are and how they are used. Maybe *server interface*?